src/App.tsx

```diff
@@ -15,71 +15,92 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+  blueLight: string,
```

Comment:
Why did you same it in the state? Try to rewrite without storing this value on the state.